config/backend-config/types.go

```diff
@@ -87,13 +87,29 @@ type ConfigT struct {
 	Sources         []SourceT       `json:"sources"`
 	Libraries       LibrariesT      `json:"libraries"`
 	ConnectionFlags ConnectionFlags `json:"flags"`
+	Settings        Settings        `json:"settings"`
 }
 
 type ConnectionFlags struct {
 	URL      string          `json:"url"`
 	Services map[string]bool `json:"services"`
 }
 
+type Settings struct {
+	DataRetention DataRetention `json:"dataRetention"`
+}
+
+type DataRetention struct {
+	EnableReportingPii     bool          `json:"enableReportingPii"`
```

Comment:
```suggestion
	EnableReportingPII     bool          `json:"enableReportingPii"`
```

https://github.com/golang/go/wiki/CodeReviewComments#initialisms

Reply:
done